src/base/bittorrent/private/resumedatasavingmanager.h

```diff
@@ -43,8 +47,9 @@ class ResumeDataSavingManager : public QObject
 
 public slots:
     void save(const QString &filename, const QByteArray &data) const;
```

Comment:
Why do you preserve this one? Is it still used?

Reply:
>Is it still used?

Yes.